readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -331,38 +331,6 @@ def sphinx_parallel_arg(self):
             return ['-j', 'auto']
         return []
 
-    def venv_sphinx_supports_latexmk(self):
```

Comment:
I know that we don't have any recent builds with Sphinx 1.6, but have we also deprecated it?

If we no longer check if `latexmk` is supported, I suppose it will fail on Sphinx<1.6 since it's not supported?

Reply:
In case there is someone building with Sphinx <1.6 (which I doubt it) it would only fail if they have the PDF enabled. Also, I'd assume that it will probably fail first with other different hundred reasons first 😄

Reply:
That seems like a fair assessment :)